microcosm_flask/swagger/schemas.py

```diff
@@ -59,14 +61,15 @@ def build(self, schema: Schema) -> Mapping[str, Any]:
         )
 
         required_fields = [
-            name
-            for name, field in fields
-            if field.required and not field.allow_none
+            name for name, field in fields if field.required and not field.allow_none
         ]
 
         if required_fields:
             result["required"] = required_fields
 
+        if schema.__doc__:
+            result["description"] = re.sub(r"\s+", " ", schema.__doc__.strip())
```

Comment:
this corrects the multiline docstring getting cut off at the `\n` ? Looks good

Reply:
It's primarily replacing extraneous whitespace in the docstring so that it looks neat in the generated documentation.